docs/how-to-guides/new-user-guides/manage-clusters/clean-cluster-nodes.md

```diff
@@ -183,7 +183,8 @@ for mount in $(mount | grep tmpfs | grep '/var/lib/kubelet' | awk '{ print $3 }'
 
 </TabItem>
 <TabItem value="RKE2">
-
+Note: for instructions for cleaning a _Non-Rancher deployed RKE2 cluster_ see: https://docs.rke2.io/install/uninstall
```

Comment:
```suggestion
:::note 
For instructions on cleaning a RKE2 cluster that weren't deployed by Rancher, see the [official RKE2 documentation](https://docs.rke2.io/install/uninstall) on uninstalling clusters.
:::
```

Reply:
Need to update the suggested change since the subject-verb agreement is off. It should be either `cleaning a RKE2 cluster that wasn't` or `cleaning RKE2 clusters that weren't`.

Reply:
Also probably worth specifying that this is about cleaning _nodes in_ clusters that weren't deployed by Rancher

Reply:
or nodes on which RKE2 was installed via Rancher

Reply:
Suggestion has been updated